app/src/main/res/layout/fragment_bookmark.xml

```diff
@@ -2,10 +2,14 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this change related to this ticket 🤔

Reply:
It makes the parent view a CoordinatorLayout to better handle snackbar interactions.